uploadservice/src/main/java/net/gotev/uploadservice/UploadTask.kt

```diff
@@ -52,7 +52,7 @@ abstract class UploadTask : Runnable {
      * Total transferred bytes. You should update this value in your subclasses when you upload
      * some data, and before calling [UploadTask.onProgress]
      */
-    private var uploadedBytes: Long = 0
+    var uploadedBytes: Long = 0
```

Comment:
why this change? I don't think we want to change the core library

Reply:
just made the variable public because it's a useful variable for calculating the progress. Alternatively I can make and use a local variable on box which would be duplicate of this.

Reply:
refrain from changing the core. this also makes this modifable. a getter would be more appropriate than changing access/scope of a private variable. make it local to you work. if anything, add a getter to a PR on the core library

Reply:
ok. Just changed it local to my code.